WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingController.java

```diff
@@ -301,6 +357,7 @@ private void sendRequestChangeEmailNotification(String oldStatus, EmailTemplateF
                         catch (javax.mail.internet.AddressException e)
                         {
                             _log.error("Error sending line item update message to " + endUser.getEmail() , e);
+                            throw new MessagingException(e.getMessage());
```

Comment:
Please pass the exception itself - chaining exceptions makes stack traces much more useful

Reply:
good point, done!